sdk/keyvault/azure-security-keyvault-jca/src/main/java/com/azure/security/keyvault/jca/AuthClient.java

```diff
@@ -135,7 +135,7 @@ private String getAccessTokenOnAppService(String resource, String identity) {
         LOGGER.entering("AuthClient", "getAccessTokenOnAppService", resource);
         LOGGER.info("Getting access token using managed identity based on MSI_SECRET");
         if (identity != null) {
-            LOGGER.log(INFO, "Using managed identity with object ID: {0}", identity);
+            LOGGER.log(INFO, "Using managed identity with client ID: {0}", identity);
```

Comment:
1. Update variable name from `identity` to `clientId`.
2. Update java doc.

Reply:
Hi, what does java doc here mean?